first-touch/src/App.js

```diff
@@ -4,13 +4,18 @@ import './App.css';
 //import Cat from 'react-cats';
 
 function App() {
-
   const [count, setCount] = useState(0);
-  let cats = [];
 
-  for (let i = 0; i < count; i++) {
-    cats.push(<li><img src="https://cataas.com/cat/gif" alt="cat"/></li>);
-   // cats.push(<Cat type = '' text = '%20' fontSize = '50' color = 'white' filter = '' width = '' height = ''/>);
+  const increment = () => {
+    if (count < 50) {
+      setCount(_currentCount => _currentCount + 1)
```

Comment:
Tu as peut-être essayé de faire des méthodes handlers pour upgrade ton `count` comme ça :
```suggestion
      setCount(count + 1)
```
Sauf que tu as un problème, `count` n'a pas la bonne valeur dans la méthode 😭  En général elle a la valeur au moment de l'initialisation de ton composant (là 0).

Dans les hooks useState, tu as deux possibilités :
1. setCount(value)
2. setCount(valeur courante  => { return "nouvelle valeur " })

La 2ᵉ possibilité te permet d'upgrade ta valeur quand tu n'as pas la valeur courante.